invokeai/frontend/web/src/features/nodes/util/graphBuilders/buildLinearTextToImageGraph.ts

```diff
@@ -48,6 +49,8 @@ export const buildLinearTextToImageGraph = (
     throw new Error('No model found in state');
   }
 
+  const onnx_model_type = model.model_type.includes('onnx');
```

Comment:
- Let's stick to camelCase unless we are dealing with variables that originated from python
- `onnx_model_type` is not particularly descriptive. Prefer something like `isUsingOnnxModel` for booleans